src/runner.js

```diff
@@ -182,10 +182,25 @@ Runner.prototype.killChild = function killChild (uuid) {
     }
 
     if (typeof child.kill === 'function') {
-        child.kill('SIGTERM');
+        var isWin = /^win/.test(process.platform);
+        var killCommand = killCommand = 'kill -9 ' + child.PhantomPID;
+
+        if (isWin) {
+            killCommand = 'taskkill /PID ' + child.PhantomPID + ' /F';
+        }
+
+        exec(killCommand, function (error, stdout, stderr) {
+            if (error !== null) {
+                console.log(chalk.bgRed(' -[ ERROR ]- '), error);
+                return;
+            }
+            child.kill('SIGTERM');
+        });
     }
 
-    this.cleanChild(uuid);
+    if (child.killed) {
```

Comment:
It's not on the Node.js docs, but I found this property get set to `true` when the child process is explicitly killed by the `kill()` command, which is what we want to do with this function.